src/allmydata/web/directory.py

```diff
@@ -45,6 +46,8 @@ class BlockingFileError(Exception):
 def make_handler_for(node, client, parentnode=None, name=None):
     if parentnode:
         assert IDirectoryNode.providedBy(parentnode)
+    if IVerifyNode.providedBy(node):
+        return VerifyNodeHandler(client, node, parentnode, name)
```

Comment:
The failure [here](https://travis-ci.org/tahoe-lafs/tahoe-lafs/jobs/208361047#L3464) probably occurs because the unknown node is a ``CiphertextFileNode``, which now implements ``IVerifyNode``. Therefore this function does not return ``UnknownNodeHandler`` as it previously did, and so the cap URI expected by the test is not included in the output.